README.md

```diff
@@ -93,7 +93,82 @@ st2::packs:
         webhook_url: XXX
 ```
 
-### Configuring Hubot (ChatOps)
+### Configuring Authentication (st2auth)
```

Comment:
This documentation describing the different auth backends is just flawless! Much better, comparing to docs.stackstorm :+1: